tests/unit/array/sort-test.js

```diff
@@ -0,0 +1,83 @@
+import { sort } from 'ember-awesome-macros/array';
```

Comment:
@kellyselden What are your thoughts on current behavior of returning `undefined` vs Ember.computed.sort behavior of returning `[]` when the source is `undefined`?